README.md

```diff
@@ -89,6 +89,30 @@ To get started, refer to our GitHub Samples [here](https://github.com/Azure/azur
   
 >[!TIP]
 > $Default [consumer group](https://learn.microsoft.com/en-us/azure/event-hubs/event-hubs-features#consumer-groups) is created by default when emulator runs. You can't create $default consumer group with supplied configuration.
+
+### Networking options
+You can run and connect to Emulator in multiple ways. Use a `Connection String` from following as per your use-case:
+
+- When the emulator container and interacting application are running natively on local machine:
+```
+"Endpoint=sb://localhost;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=SAS_KEY_VALUE;UseDevelopmentEmulator=true;"
```

Comment:
Can we use this exact string (with the "UseDevelopmentEmulator=true") at the end in our .net sample code too?